coalib/output/ConsoleInteraction.py

```diff
@@ -649,88 +649,106 @@ def show_enumeration(console_printer,
     console_printer.print()
 
 
-def show_bear(console_printer, bear, sections, metadata):
+def show_bear(bear,
+              sections,
+              show_description,
+              show_params,
+              console_printer):
     """
     Display all information about a bear.
 
-    :param console_printer: Object to print messages on the console.
-    :param bear:            The bear to be displayed.
-    :param sections:        The sections to which the bear belongs.
-    :param metadata:        Metadata about the bear.
-    """
-    console_printer.print(bear.name + ":")
-    console_printer.print("  " + metadata.desc.replace("\n", "\n  ") + "\n")
-
-    show_enumeration(
-        console_printer, "Supported languages:",
-        bear.supported_languages,
-        "  ",
-        "The bear does not provide information about which languages "
-        "it can analyze.")
-    show_enumeration(console_printer,
-                     "Used in:",
-                     sections,
-                     "  ",
-                     "No sections.")
-
-    show_enumeration(console_printer,
-                     "Needed Settings:",
-                     metadata.non_optional_params,
-                     "  ",
-                     "No needed settings.")
-    show_enumeration(console_printer,
-                     "Optional Settings:",
-                     metadata.optional_params,
-                     "  ",
-                     "No optional settings.")
-
-
-def print_bears(console_printer, bears, compress):
+    :param bear:             The bear to be displayed.
+    :param sections:         The sections to which the bear belongs.
```

Comment:
It'd be nice to mention the data structure for `sections`. I believe it's list here?